node/config.ts

```diff
@@ -121,6 +124,12 @@ const logConfigError = (func: EdgeFunction, exitCode: number, stderr: string, lo
 
       break
 
+    case ConfigExitCode.InvalidDefaultExport:
+      errMsg = `Default export in edge function at '${func.path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api`
```

Comment:
Do we need to initialise this variable at the top and mutate it here? Could we instead just send the string directly to the `Error` constructor?